app/src/main/java/com/avans/rentmycar/ui/offer/OfferCarFragment.kt

```diff
@@ -233,7 +231,7 @@ class OfferCarFragment : Fragment(), DatePickerDialog.OnDateSetListener, TimePic
     override fun onDateSet(view: DatePicker?, year: Int, month: Int, dayOfMonth: Int) {
         Log.d(TAG, "onDateSet()" )
         saveDay = dayOfMonth
-        saveMonth = month
+        saveMonth = month + 1
```

Comment:
dit is interessant, maar als het werkt..

Reply:
Ik beken dat ik niet heel de code van de OfferCarFragment uitgeplozen heb, maar slechts ben afgegaan op wat Aubrey in Teams zei en toen werkte het…